src/jobs/jobs.service.ts

```
@@ -66,6 +75,14 @@ export class JobsService {
 
   // Application CRUD
   async createApplication(createApplicationDto: CreateApplicationDto): Promise<Application> {
```

ESLint

[error] Replace `createApplicationDto:·CreateApplicationDto` with `⏎····createApplicationDto:·CreateApplicationDto,⏎··`

(prettier/prettier)